server/android/src/android.rs

```diff
@@ -33,6 +33,7 @@ pub mod android {
                     &Record::builder()
                         .args(format_args!("{}", record.args))
                         .target("omSupply")
+                        .module_path(Some("omSupply"))
```

Comment:
this will set the `tag` to be `omSupply` in the logcat. Makes for easier filtering of events.